Doc/update (#351)
vtex-apps/product-summary: docs/ProductSummaryShelf.md

# Product Summary Shelf

Product Summary Shelf is the main block exported by the [Product Summary app](https://developers.vtex.com/vtex-developer-docs/docs/vtex-product-summary). Through its child blocks, it provides the needed structure to render the Product Summary component in your store.

![](https://user-images.githubusercontent.com/40380674/96649443-7d21d480-1307-11eb-9100-534fa9e70ca6.png))

## Configuration

1. Import the `vtex.product-summary` app to your theme's dependencies in the `manifest.json`:

```json
  dependencies: {
    "vtex.product-summary": "2.x"
  }
```

> 2. Add the `product-summary.shelf` block as child of the [`list-context.product-list` block](https://developers.vtex.com/vtex-developer-docs/docs/vtex-product-summary-productsummarylist#product-list-block) :

Reviewer: ```suggestion
2. Add the `product-summary.shelf` block as a child of the [`list-context.product-list` block](https://developers.vtex.com/vtex-developer-docs/docs/vtex-product-summary-productsummarylist#product-list-block) :
```

}
```

> ⚠️ 
> 

> > Do not forget to also declare the other Product Summary's blocks in your theme, otherwise the component won't be displayed. 

Reviewer: ```suggestion
> Remember to define the other Product Summary blocks in your theme as well, otherwise the component will not be displayed.
```
